terraform/modules/enclave/prometheus/prometheus.conf.tpl

```diff
@@ -0,0 +1,24 @@
+global:
+  scrape_interval:     1s
```

Comment:
Do we want these to be so aggressive? Default is 1m, our PaaS prometheus is every 30 seconds.

Reply:
We have reverted the scrape interval to 30 seconds